jenkins_jobs/build_host_os/archive.sh

```diff
@@ -15,4 +15,4 @@ rsync_upload --recursive $JENKINS_BUILD_ARCHIVE_DIR_NAME
 
 rsync_upload SUCCESS
 rsync_upload BUILD_TIMESTAMP
-rsync_upload --recursive repository
+rsync_upload --recursive repository/packages/latest/*
```

Comment:
Having a directory named "repository" at this level makes no sense anymore. We could just leave the default results directory in `build_host_os/script.sh`. The `createrepo` command in that file will also need to change to `createrepo repository/packages/latest`.

This command will upload the packages and repodata directories flat to the archive directory, at the same level of "SUCCESS" and "BUILD_TIMESTAMP". I suggest:
```
ln -s result/packages/latest repository
rsync_upload --recursive --copy-links repository
```
So that we keep the same archive structure as before:
```
SUCCESS
BUILD_TIMESTAMP
repository/
    wok/
    kernel/
    repodata/
```